facebook_business/api.py

```diff
@@ -848,7 +848,8 @@ def load_next_page(self):
 
         if 'paging' in response and 'next' in response['paging']:
             self._path = response['paging']['next']
-            self.params = {}
+            # Comment line below because of the params bug present in FB end
+            # self.params = {}
```

Comment:
We want to remove from it only the keys that are in `response['paging']['next']`

For example:

```python
params = {"thumbnail_width": 1024, "thumbnail_height": 1024}

from urllib.parse import parse_qs, urlparse

url = "https://graph.facebook.com/v12.0/act_796876917805027/adcreatives?access_token=THE_TOKEN&pretty=0&fields=account_id%2Cobject_type%2Cname%2Ctitle%2Cbody%2Cimage_url%2Cimage_hash%2Cvideo_id%2Ccall_to_action_type%2Cobject_url%2Cthumbnail_url%2Cobject_story_id%2Cobject_story_spec%2Casset_feed_spec%2Cinstagram_permalink_url%2Ceffective_object_story_id%2Cproduct_set_id&summary=true&limit=25&after=MjM4NTAwNjc2OTE5MzA2MDcZD"

parsed_url = urlparse(url)
parsed_qs = parse_qs(parsed_url.query)

keys_in_pagination_url = list(parsed_qs.keys())

for key in keys_in_pagination_url:
    params.pop(key, None)
```

Reply:
I tried something similar

Reply:
Nice